Yosemite/YosemiteTests/Stores/Order/OrdersUpsertUseCaseTests.swift

```diff
@@ -102,6 +123,23 @@ final class OrdersUpsertUseCaseTests: XCTestCase {
 }
 
 private extension OrdersUpsertUseCaseTests {
+
+    func sampleOrderItem(itemID: Int64, taxes: [Networking.OrderItemTax]) -> Networking.OrderItem {
```

Comment:
nit: maybe a name that is consistent with `makeOrder` below?
```suggestion
    func makeOrderItem(itemID: Int64, taxes: [Networking.OrderItemTax]) -> Networking.OrderItem {
```